lib/pf/Switch/Fortinet/FortiGate.pm

```diff
@@ -48,7 +48,7 @@ use pf::SwitchSupports qw(
     WirelessMacAuth
     WiredMacAuth
     WirelessDot1x
-    RoleBasedEnforcement
```

Comment:
RoleBasedEnforcement should be supported as well because of WSSO. https://docs.fortinet.com/document/fortiap/6.4.0/fortiwifi-and-fortiap-cookbook/414919/wifi-with-wsso-using-windows-nps-and-user-groups